lib/commonmark.js

```diff
@@ -86,7 +86,7 @@ function renderHtml(markdownString, options, cb) {
 }
 
 function parse(markdownString, cb) {
-  return cb(null, commonmarkRenderer.parse(markdownString));
+  return cb(null, commonmarkRenderer.parse(markdownString, {}));
```

Comment:
Should I change the interface and make a breaking change?
Even if I would change the name it would be a breaking change anyway.